src/components/mini-cart/mini-cart.hbs

```diff
@@ -3,7 +3,7 @@
     <button class="mini-cart__btn--close" data-js="mini-cart-close">
         <span class="sr-only">Close</span>
         <svg class="icon--close-btn">
-            <use xlink:href="../sprite.svg#close"></use>
+            <use xlink:href="{{rootPath}}/sprite.svg#close"></use>
```

Comment:
Should this be `{{../rootPath}}` too? I didn't know you can do this, but I'm only asking because mini-cart.hbs and mini-cart-items.hbs are in the same folder.

Reply:
no it shouldn't here. the HB variables are relative to the object that they are in, so the items within mini-cart-items is being iterated over, while the rootPath is a sibling to those items, hence the need to ../ back to that node.
In mini-cart, it is at the same level so no ../ is needed.